site/docs/patterns/header-block.mdx

```diff
@@ -0,0 +1,148 @@
+- Maintain a visually consistent layout and presentation for headers, and establish visual hierarchy of headings with supplementary text across different components and content layouts.
+- Display headings with status icons to communicate different statuses, depending on the context of the component the header block sits within.
+
+## How to build
+
+### Anatomy
+
+The header block pattern comprises the following visual and text elements:
+
+1. **Accent bar**: spans the height of, and brings attention, to the heading and supplementary text.
+2. **Heading**: serves as the primary title of a content type.  
+3. **Pre-header**: displayed above the main heading. Used to introduce a section or draw attention to important information, such as a grouping or step in a process.  
+4. **Description**: displayed below the main heading as a secondary text element that provides additional information or context to the main title. Brief descriptions can be used to clarify the main title, or provide further details about the content of the design.
+5. **Close button**: optional action for the user to dismiss a component or element.  
+6. **Status icon**: visual indicator that communicates status.
+
+<Diagram
+  src="/img/patterns/header-block/header-block-anatomy.png"
+  alt="A header block anatomy."
+  border
+/>
+
+### Layout
+
+Header block can be configured to be displayed using three different sizes, each differentiated by the size of the padding and the heading type. We recommend using an appropriately sized header block, relative to the container element it is embedded within.
+
+As a rule of thumb, a large container in Salt uses padding size of `--salt-spacing-300`, a medium container uses `--salt-spacing-200`, and a small container uses `--salt-spacing-100`. For example, the Dialog component uses `--salt-spacing-300` padding between elements, therefore embedding a header block also using `--salt-spacing-300` maintains a consistent layout and ensures all elements align as expected.
+
+<Diagram
+  src="/img/patterns/header-block/header-block-layout-spacing.png"
+  alt="A header block layout."
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Header block with `--salt-spacing-100` padding does not contain an accent bar. We recommend using this header block size within components where visual emphasis is not important. Consider the available space for an accent bar, relative to the size of the component.
+</ExampleContainer>
+
+## Layout variations
+
+A header block has a number of possible layout variations depending on the amount of information you want to communicate, the user workflow and the container it will sit in.
+
+### With accent bar and all text elements
+
+Display headings with an accent bar which draws attention to the heading, reinforces brand identity, and remains consistent with Salt Design System’s theming options.
```

Comment:
"Display headings", confused me. I was trying to decided whether this was display in terms of typography or a command.  I think the sentence needs updating